java-logging-appinsights/src/main/java/uk/gov/hmcts/reform/logging/appinsights/autoconfiguration/AppInsightsAutoConfiguration.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.logging.appinsights.autoconfiguration;
+
+import org.springframework.context.annotation.Bean;
+import uk.gov.hmcts.reform.logging.appinsights.telemetry.initializers.ContextVersionInitializer;
+
+/**
+ * <h1>Core Application Insights Configuration</h1> .
+ *
+ * <p>This class provides the custom Configuration for AppInsights .</p>
+ */
+public class AppInsightsAutoConfiguration {
+
+    /**
+     * Bean for ContextVersionInitializer.
+     *
+     * @return instance of {@link ContextVersionInitializer}
+     */
+    @Bean
+    public ContextVersionInitializer contextVersionInitializer() {
+        return new ContextVersionInitializer();
+    }
+
+}
```

Comment:
Given the comment about AppInsights config this will become retired

Reply:
The idea was to make them all initialised as spring beans rather than using  xml based configs. I was pretty much planning to get rid of other 3 initialisers as well and move to Java config so that we are consistent with the starter and I thought we would want to move away from  xml configs?

Reply:
Forgive me for confusion, let's move away from xml completely. I was too Microsoft-y about all this

Reply:
Sure, will get rid of the others as well.

Reply:
Done, got rid of XML configs